homeassistant/components/lg_thinq/strings.json

```diff
@@ -993,6 +993,28 @@
           "normal": "[%key:component::lg_thinq::entity::sensor::current_dish_washing_course::state::delicate%]"
         }
       }
+    },
+    "water_heater": {
+      "water_boiler": {
+        "name": "Hot water",
+        "state_attributes": {
+          "current_temperature": {
+            "name": "Hot water temperature"
+          }
+        }
```

Comment:
isn't this already the case?

Reply:
This one is still open

Reply:
The base water_heater platform already provides translation for the `current_temperature` attribute, so I also don't think this is needed